### ISSUE TITLE
[Bug]: Error: Cannot find module 'node:https'

### ISSUE THREAD

**Issue by antoniokly** [homebridge-eufy-security/plugin#145]
### What happened?

Error happened on loading the plugin. It used to work on a previous version, but not sure which version I upgraded from.

### Device Type

Indoor Camera (Supported)

### Plugin Version

v2.0.x (Supported)

### HomeBridge Version

v1.4.x (Supported)

### NodeJS Version

v14 (Supported)

### Operating System

Raspbian 10+ (Supported)

### Relevant log output

```shell
[18/08/2022, 12:24:14 am] ERROR LOADING PLUGIN homebridge-eufy-security:
[18/08/2022, 12:24:14 am] Error: Cannot find module 'node:https'
Require stack:
- /usr/local/lib/node_modules/homebridge-eufy-security/dist/controller/SnapshotManager.js
- /usr/local/lib/node_modules/homebridge-eufy-security/dist/controller/streamingDelegate.js
- /usr/local/lib/node_modules/homebridge-eufy-security/dist/accessories/CameraAccessory.js
- /usr/local/lib/node_modules/homebridge-eufy-security/dist/platform.js
- /usr/local/lib/node_modules/homebridge-eufy-security/dist/index.js
- /usr/local/lib/node_modules/homebridge/lib/plugin.js
- /usr/local/lib/node_modules/homebridge/lib/pluginManager.js
- /usr/local/lib/node_modules/homebridge/lib/server.js
- /usr/local/lib/node_modules/homebridge/lib/cli.js
- /usr/local/lib/node_modules/homebridge/bin/homebridge
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:889:15)
    at Function.Module._load (internal/modules/cjs/loader.js:745:27)
    at Module.require (internal/modules/cjs/loader.js:961:19)
    at require (internal/modules/cjs/helpers.js:92:18)
    at Object.<anonymous> (/usr/local/lib/node_modules/homebridge-eufy-security/src/plugin/controller/SnapshotManager.ts:1:1)
    at Module._compile (internal/modules/cjs/loader.js:1072:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1101:10)
    at Module.load (internal/modules/cjs/loader.js:937:32)
    at Function.Module._load (internal/modules/cjs/loader.js:778:12)
    at Module.require (internal/modules/cjs/loader.js:961:19)
[18/08/2022, 12:24:14 am] ====================
```

**Comment by thieren** [comment]
Hi @antoniokly 

can you check your node version? I believe you have a version prior to 14.18. If you update to the newest (for v14 that would be 14.20 I think) this should be fine.

**Comment by thieren** [comment]
I've fixed this in the RC v2.1.0

**Comment by thieren** [comment]
Is fixed with release of 2.1.0
